hyppo/discrim/discrim_one_samp.py

```diff
@@ -1,21 +1,14 @@
-from typing import NamedTuple
-
 import numpy as np
 from scipy._lib._util import MapWrapper
 
 from ._utils import _CheckInputs
 from .base import DiscriminabilityTest
 
 
-class DiscrimOneSampleTestOutput(NamedTuple):
```

Comment:
> I did pull the latest changes from master to my local repository before making changes using git pull upstream master after I forked it 2 days ago. The branch 'goodness_of_fit' now has a kgof folder in hyppo/tools. Can you elaborate on how the changes are reverting new additions?

This is one example of what I was talking about. We added a `NamedTuple` for outputs recently, but those changes aren't included in the new file. Same with a few more changes in these files.